### DISCUSSION TITLE
How to  change the generate output files path ?

### DISCUSSION THREAD

**Discussion by sea-lanya** [nuxt/nuxt#16480] [Questions]
default is  .outout/public

But I  want change to  ../server/www   etc..

**Comment by TheAlexLichter** [comment]
You can change it via [`nitro.output`](https://v3.nuxtjs.org/api/configuration/nuxt-config#nitro) in your `nuxt.config.ts`. Check [the Nitro docs](https://nitro.unjs.io/config#output) for more.

For your use case, you could also copy the files over after the build process instead of changing the output dir ☺️

**Reply by sea-lanya** [reply]
@manniL   thanks.

I tried.
But I Can't  set  the path  out of the project's path.

config:
```
nitro: {
        output: {
            publicDir: ".server/www"
        }
    },
```

```
projectDir/public/xxx    done
```

config:
```
nitro: {
        output: {
            publicDir: "./../server/www"
        }
    },
```

```
projectDir/xxx
public/xxx     not work...
```
